bucket/markdown-monster.json

```
@@ -0,0 +1,26 @@
+{
+    "homepage": "http://markdownmonster.west-wind.com",
+    "description": "Markdown editing and weblog publishing tool.",
+    "license": "Shareware",
+    "version": "1.19.15",
+    "url": "https://west-wind.com/files/MarkdownMonsterPortable.zip",
+    "hash": "2fd99fcbcf8d71eb4f3b25a014fa8e11463a5e938aa9573f0f36b461f7b84efa",
+    "bin": [
+        "mm.exe",
+        "MarkdownMonster.exe"
+    ],
+    "shortcuts": [
+        [
+            "MarkdownMonster.exe",
+            "Markdown Monster"
+        ]
+    ],
+    "persist": "PortableSettings",
+    "checkver": {
+        "url": "http://markdownmonster.west-wind.com/download.aspx",
+        "regex": "<(?:V|v)ersion>([\\d.]+)</(?:V|v)ersion>"
```

If you insist on both cases

```suggestion
        "regex": "<[vV]ersion>([\\d.]+)</[vV]ersion>"
```